### PR TITLE
OKTA-742877 - Update ITP event type names

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** As per OKTA-745429, update event type names 
- **Is this PR related to a Monolith release?** 2024.08.0

### Resolves:

* [OKTA-742877](https://oktainc.atlassian.net/browse/OKTA-742877)

### CODE REVIEW

**Review by brentschaus-okta — DISMISSED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md [hunk 1]
```diff
@@ -73,10 +73,51 @@ This resource contains detailed reference material on event types triggered with
...
+| **event.system.debugContext.debugData**                |                 |         |         |
+| Behaviors             | List of behaviors identified for the current event. `POSITIVE` - the specific behavior is identified. `NEGATIVE` - the specific behavior wasn't identified. See [About Behavior Detection](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-behavior-detection).                 | key-value pairs        |  `{New Geo-Location=POSITIVE, New Device=NEGATIVE, New IP=POSITIVE, New State=POSITIVE, New Country=POSITIVE, Velocity=POSITIVE, New City=POSITIVE} `       |
+| Risk            | Contains the level of risk for a particular request (`LOW`, `MEDIUM`, or `HIGH`) and the `reasons` that contributed to the risk level.                | key-value pairs         |`{reasons=Anomalous Geo-Distance, New Device, New ASN, New IP, New State, New Country, New City, level=HIGH}`          |
+| ThreatSuspected            | If ThreatInsight is running and detects a request as suspicious, the value for this property is `true`.                    | Boolean         | `false`         |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md:86]:
```suggestion
| ThreatSuspected            | If ThreatInsight is running and detects a suspicious request, the value for this property is `true`.                    | Boolean         | `false`         |
```

File: packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md [hunk 2]
```diff
@@ -73,10 +73,51 @@ This resource contains detailed reference material on event types triggered with
...
+| **target** (Rule)         | The rule associated with the Continuous Access evaluation           | Object     |        |
+| type       | The type of target object           | String     | Rule       |
+| **target.DetailEntry** (Rule)       |             |      |        |
+| RuleAction        | The configured action to respond to the risk. Values include `TERMINATE_SESSION` or `RUN_WORKFLOW`.            | Enum     | `TERMINATE_SESSION`       |
+| SingleLogOutEnabled        | For a `RuleAction` of `TERMINATE_SESSION`, and if `true`, a Continuous Access evaluation violation enforces application logout             | Boolean     | `true`      |
+| SingleLogOutSelectionMode        | For a `RuleAction` of `TERMINATE_SESSION`, the options of the application logout, either all applications, specific applications, or none. Values can be: `NONE`, `ALL`, or `SPECIFIED`.           | Enum     | `ALL`       |
+| WorkflowId         | The unique identifier of the workflow if the `RuleAction` is `RUN_WORKFLOW`.           | String     | 572749       |
+| DisplayName        | Displays the name of the rule           | String     | Entity Risk Policy       |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md:97]:
```suggestion
| DisplayName        | The name of the rule           | String     | Entity Risk Policy       |
```

File: packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md [hunk 3]
```diff
@@ -73,10 +73,51 @@ This resource contains detailed reference material on event types triggered with
...
+| PolicyAction        | The configured action to respond to the risk. Values include `TERMINATE_SESSION` or`RUN_WORKFLOW`.              | Enum     | `TERMINATE_SESSION`       |
+| PolicySingleLogOutEnabled        |  For a `PolicyAction` of `TERMINATE_SESSION`, and if `true`, a continuous access evaluation violation enforces application logout             | Boolean     | `true`      |
+| PolicySingleLogOutSelectionMode        | For a `PolicyAction` of `TERMINATE_SESSION`, the options of the application logout, either all applications, specific applications, or none. Values can be: `NONE`, `ALL`, or `SPECIFIED`.           | Enum     | `ALL`       |
+| PolicySingleLogoutAppInstanceIds          | A list of apps that will that will be logged out if the `PolicySingleLogOutMode` mode is `SPECIFIED`.            | Array     | `[ "0oa1gkh63g214r0Hq0g4", "0oa1gjh63g214q0Iq3g3" ]`      |
+| WorkflowId         | The unique identifier of the workflow if the `PolicyAction` is `RUN_WORKFLOW`.           | String     | 572749       |
+| DisplayName        | Displays the name of the action          | String     | `TERMINATE_SESSION`       |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md:109]:
```suggestion
| DisplayName        | The name of the action          | String     | `TERMINATE_SESSION`       |
```

Comment (grahamsmith-okta) [reply packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md:109]: Reason I'm changing these is that none of the others begin with a verb

File: packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md [hunk 4]
```diff
@@ -73,10 +73,51 @@ This resource contains detailed reference material on event types triggered with
...
+
+| Key event properties  | Description                                         | Data type      | Example values |
+| --------------------- | --------------------------------------------------- | -------------- | -------------- |
+| **event.system.debugContext.debugData**                |                 |         |         |
+| Behaviors             | List of behaviors identified for the current event. `POSITIVE` - the specific behavior is identified. `NEGATIVE` - the specific behavior wasn't identified. See [About Behavior Detection](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-behavior-detection).                 | key-value pairs        |  `{New Geo-Location=POSITIVE, New Device=NEGATIVE, New IP=POSITIVE, New State=POSITIVE, New Country=POSITIVE, Velocity=POSITIVE, New City=POSITIVE} `       |
+| Risk            | Contains the level of risk for a particular request (`LOW`, `MEDIUM`, or `HIGH`) and the `reasons` that contributed to the risk level.                | key-value pairs         |`{reasons=Anomalous Geo-Distance, New Device, New ASN, New IP, New State, New Country, New City, level=HIGH}`          |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md:85]:
```suggestion
| Risk            | The level of risk for a particular request (`LOW`, `MEDIUM`, or `HIGH`) and the `reasons` that contributed to the risk level.                | key-value pairs         |`{reasons=Anomalous Geo-Distance, New Device, New ASN, New IP, New State, New Country, New City, level=HIGH}`          |
```

File: packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md [hunk 5]
```diff
@@ -143,6 +184,37 @@ This resource contains detailed reference material on event types triggered with
...
+
+| Key event properties | Description                                         | Data type      | Example values |
+| --------------------- | --------------------------------------------------- | -------------- | -------------- |
+| **event.system.debugContext.debugData**                |                 |         |         |
+| Behaviors             | List of behaviors identified for the current event. `POSITIVE` - the specific behavior is identified. `NEGATIVE` - the specific behavior wasn't identified. See [About Behavior Detection](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-behavior-detection).                 | key-value pairs        |  `{New Geo-Location=POSITIVE, New Device=NEGATIVE, New IP=POSITIVE, New State=POSITIVE, New Country=POSITIVE, Velocity=POSITIVE, New City=POSITIVE} `       |
+| Risk            | Contains the level of risk for a particular request (`LOW`, `MEDIUM`, or `HIGH`) and the `reasons` that contributed to the risk level.                | key-value pairs         |`{reasons=Anomalous Geo-Distance, New Device, New ASN, New IP, New State, New Country, New City, level=HIGH}`          |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md:197]:
```suggestion
| Risk            | The level of risk for a particular request (`LOW`, `MEDIUM`, or `HIGH`) and the `reasons` that contributed to the risk level.                | key-value pairs         |`{reasons=Anomalous Geo-Distance, New Device, New ASN, New IP, New State, New Country, New City, level=HIGH}`          |
```

File: packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md [hunk 6]
```diff
@@ -143,6 +184,37 @@ This resource contains detailed reference material on event types triggered with
...
+| ThreatSuspected            | If ThreatInsight is running and detects a request as suspicious, the value for this property is `true`.                   | Boolean         | `false`         |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md:198]:
```suggestion
| ThreatSuspected            | If ThreatInsight is running and detects a suspicious request, the value for this property is `true`.                   | Boolean         | `false`         |
```

File: packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md [hunk 7]
```diff
@@ -143,6 +184,37 @@ This resource contains detailed reference material on event types triggered with
...
+| type        | The type of target object     | String     | User       |
+| **target** (Rule)         | The rule associated with the Continuous Access evaluation            | Object     |        |
+| type        | The type of target object           | String     | Rule       |
+| **target.DetailEntry** (Rule)       |             |      |        |
+| RuleAction        | The configured action to respond to the risk. Values include `TERMINATE_SESSION` or `RUN_WORKFLOW`.             | Enum     | `TERMINATE_SESSION`       |
+| SingleLogOutEnabled        | For a `RuleAction` of `TERMINATE_SESSION`, and if `true`, a Continuous Access evaluation violation enforces app logout.          | Boolean     | `true`      |
+| SingleLogOutSelectionMode        | For a `RuleAction` of `TERMINATE_SESSION`, the options of the app logout, either all apps, specific apps, or none. Values can be: `NONE`, `ALL`, or `SPECIFIED`.           | Enum     | `ALL`       |
+| WorkflowId         | The unique identifier of the workflow if the `RuleAction` is `RUN_WORKFLOW`.           | String     | 572749       |
+| DisplayName        | Display the name of the target rule           | String     | Test Rule      |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md:209]:
```suggestion
| DisplayName        | The name of the target rule           | String     | Test Rule      |
```

**Review by grahamsmith-okta — DISMISSED** [review]
A few comments for consistent structure

**Review by barbaravo-okta — APPROVED** [review]

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**